airflow/models/dag.py

```diff
@@ -2435,6 +2436,56 @@ def cli(self):
         args = parser.parse_args()
         args.func(args, self)
 
+    def test(self, execution_date: datetime | None = None, run_conf=None, session=None):
+        """Execute one single DagRun for a given DAG and execution date, using the DebugExecutor."""
+
+        def add_logger_if_needed(ti: TaskInstance):
```

Comment:
for some reason, log messages emitted from within `_run_raw_task` are not forwarded to stdout.  the only case where this is done is when the task is deferred, so in that case, the user does not see that message and then the task is just marked successful.  i don't think you need to fix the fact that triggers are currently unsupported ... but.... if it's possible to at least bubble up the message, then that's good.  not sure how easy that would be though.  

probably _best_ thing to do is fail the whole operation in this case!  and i have a thought on that i'll comment below....

Reply:
@dstandish I fixed this so that the logs now show up. Can you try again and lemme know if this works for you?

Reply:
yup looks good